css/properties.json

```diff
@@ -2792,7 +2792,7 @@
     "status": "experimental"
   },
   "content": {
-    "syntax": "normal | none | [ &lt;string&gt; | &lt;uri&gt; | &lt;counter&gt; | attr() | open-quote | close-quote | no-open-quote | no-close-quote ]+",
+    "syntax": "normal | none | [ &lt;string&gt; | &lt;url&gt; | &lt;counter&gt; | attr() | open-quote | close-quote | no-open-quote | no-close-quote ]+",
```

Comment:
FYI, CSS 2.2 still [defines it using `<uri>`](https://www.w3.org/TR/CSS22/generate.html#propdef-content), but [it was resolved](https://lists.w3.org/Archives/Public/www-style/2016Oct/0091.html) to refer to [CSS Syntax](https://drafts.csswg.org/css-syntax/) instead, which removed `<uri>`. So it's fine that it gets changed already.

Sebastian